server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/configuration/ConfigurationChangedDTO.java

```diff
@@ -63,14 +65,16 @@ static ConfigurationChangedDTO from(ConfigurationChanged configurationChanged) {
         @JsonProperty("type") String type,
         @JsonProperty("bucketCount") int bucketCount,
         @JsonProperty("updateBrowseStartPace") int updateBrowseStartPace,
-        @JsonProperty("sliceWindow") Duration sliceWindow) {
+        @JsonProperty("sliceWindow") Duration sliceWindow,
+        @JsonProperty("sizeMetricsEnabled") boolean sizeMetricsEnabled) {
```

Comment:
Changing the persisted object inside the EventStore should raise a big warning sign (we clearly don't want to handle now the complexity of that topic)